include/zephyr/net/wifi.h

```diff
@@ -14,6 +14,13 @@
 
 #define WIFI_COUNTRY_CODE_LEN 2
 
+/* As per hardware design for power efiicienecy
+ * minimum listen interval period should be
+ * 3 DTIM period.
+ */
+#define WIFI_LISTEN_INTERVAL_MIN 3
```

Comment:
As this is generic, we should follow protocol limits, any HW specific limits should be enforced by the underlying driver.